_posts/2023-07-20-alert-anomaly-visual.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "Visualizing alerts and anomalies using OpenSearch Dashboards visualization tools and applications"
+authors: 
+  - jdbright
+  - ashisagr
+  - ohltyler
+  - amgalitz
+  - hnyng
+  - xeniatup
+date: 2023-07-20
+meta_keywords: alerting, anomaly detection, alert, anomaly, observabilty, dashboard, dashboards, visualizations, visualize
+meta_description: Integrating alerts and anomalies visualizations across OpenSearch Dashboards applications.
```

Comment:
"Integrating alerts and anomalies visualizations across OpenSearch Dashboards applications." sounds confusing to me, maybe "Integrating alerts and anomalies with OpenSearch Dashboard visualizations"?